src/Pos/Wallet/Web/Server/Sockets.hs

```diff
@@ -91,5 +101,54 @@ type WebWalletSockets m = (MonadWalletWebSockets m, MonadIO m)
 runWalletWS :: ConnectionsVar -> WalletWebSockets m a -> m a
 runWalletWS = flip Ether.runReaderT
 
-notify :: WebWalletSockets m => NotifyEvent -> m ()
-notify msg = getWalletWebSockets >>= flip sendWS msg
+notifyAll :: WebWalletSockets m => NotifyEvent -> m ()
+notifyAll msg = do
+  var <- getWalletWebSockets
+  conns <- readTVarIO var
+  for_ (listConnections conns) $ flip sendWS msg
+
+------------------------------------
+-- Implementation of ConnectionMap
+------------------------------------
+
+data IntMapWithUnusedKey v = IntMapWithUnusedKey
+    { unusedKey :: Int
+    , mapping   :: IntMap v
```

Comment:
We have convection for fields to prefix them with first letters of type name.
I guess it would be hard (or funny) to follow in this case, but still. Maybe simplify type's name (ala `CountingMap` or smth)?